### DISCUSSION TITLE
How to get CLI access into my router to install this?

### DISCUSSION THREAD

**Discussion by maticrepse** [Salamek/huawei-lte-api#175] [Q&A] [answered]
I found this repository by looking for ways to forward SMS's that the SIM receives. How do I even get CLI access to the router, does some other type of firmware need to be installed?

My device (B535-232) is currently running the configuration below:
Hardware version: WL1B535M
Software version: 10.0.1.1(H191SP4C983)
Web UI version: WEBUI 10.0.1.1(W2SP3C21)
Configuration file version: B535-232-CUST 10.0.1.2(C21)

**Comment by Salamek** [comment, marked as answer]
You don't install this to your router, you install this to your PC/Raspberry/etc. this library communicates with your router...

**Reply by maticrepse** [reply]
Aaah, I feel stupid now lmao. I don't know why, but it made a lot of sense to me, that you could run some kind of a unix OS on it that could have python installed etc.. Thank you for the explanation!
